main.go

```diff
@@ -277,6 +277,37 @@ func getGeneralRemunerationFromYear(c echo.Context) error {
 	return c.JSON(http.StatusOK, data)
 }
 
+func getAllAgencies(c echo.Context) error {
+	agencies, err := client.Db.GetAllAgencies()
+	if err != nil {
+		return c.JSON(http.StatusInternalServerError, "Error while listing agencies")
+	}
+	return c.JSON(http.StatusOK, agencies)
+}
+
+func getAgencyById(c echo.Context) error {
+	agencyName := c.Param("orgao")
+	agency, err := client.Db.GetAgency(agencyName)
+	if err != nil {
+		return c.JSON(http.StatusNotFound, "Agency not found")
+	}
+	return c.JSON(http.StatusFound, agency)
+}
+
+func getMonthlyInfo(c echo.Context) error {
+	year, err := strconv.Atoi(c.Param("ano"))
+	if err != nil {
+		return c.JSON(http.StatusBadRequest, fmt.Sprintf("Parâmetro ano=%d inválido", year))
+	}
+	agencyName := c.Param("orgao")
```

Comment:
Adiciona o parâmetro mês e se ele for vazio, não passa. Daí não precisamos criar outro endpoint, caso necessário